### PR TITLE
Cleaning up peer review files and updating story

### PR DESCRIPTION
**Pull request by hannahbergam**
PeerReviewLinkSection is an interesting storybook case. First of all, this file was refactored 7 years ago ([PR here](https://github.com/code-dot-org/code-dot-org/pull/18278/files)) and the storybook was not cleaned accordingly. Noteworthy: the filterType prop was removed. This means there is no longer a difference between what the first and second stories looked like. Additionally, the component was taking in an escalated id prop but not using it, so I removed that prop from there and the one other component that calls it. 

Keeping these stories because we don't generally have a ton of insight into this side of the world (as evidenced by it not being updated 7 years later) so keeping the scant coverage we have. 

Screenshot 2024-01-30 at 1 59 04 PM

Screenshot 2024-01-30 at 2 21 11 PM

Screenshot 2024-01-30 at 3 39 11 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1431

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/code-studio/peer_reviews/PeerReviewLinkSection.jsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import moment from 'moment';
 class PeerReviewLinkSection extends React.Component {
   static propTypes = {
     reviews: PropTypes.arrayOf(PropTypes.array).isRequired,
-    escalatedReviewId: PropTypes.number,
```
Comment (hannahbergam) [inline apps/src/code-studio/peer_reviews/PeerReviewLinkSection.jsx:9]: The file never calls this prop...

File: apps/src/code-studio/pd/workshop_dashboard/components/survey_results/free_response_section.story.jsx [hunk 2]
```diff
@@ -1,47 +1,43 @@
 import React from 'react';
 import FreeResponseSection from './free_response_section';
-import reactBootstrapStoryDecorator from '../../../reactBootstrapStoryDecorator';
```
Comment (hannahbergam) [inline apps/src/code-studio/pd/workshop_dashboard/components/survey_results/free_response_section.story.jsx:3]: This decorator is just a div wrapping the component with an id, so I did this manually below (decorators are not supported in the new storybook)

File: apps/src/code-studio/peer_reviews/PeerReviewLinkSection.story.jsx [hunk 3]
```diff
@@ -1,40 +1,23 @@
...
-              [4, ''],
-            ]}
-          />
-        ),
-      },
-      {
-        name: 'Peer Review Links for escalated submissions',
-        story: () => (
-          <PeerReviewLinkSection
-            filterType="escalated"
```
Comment (hannahbergam) [inline apps/src/code-studio/peer_reviews/PeerReviewLinkSection.story.jsx:28]: filterType and escalatedReviewId are no longer possible props, so we don't need this second story!

**Review by TurnerRiley — APPROVED** [review]
Thank you for doing extra cleanup with the props we no longer need!

**Pull request merged**